### ISSUE TITLE
Flow simulation in a forest

### ISSUE THREAD

**Issue by rakyuv** [firemodels/fds#12549]
**Description**
I am interested to simulate air flow over a forest. Thus I would like to see the deceleration of air at the level of trees and free stream velocity above the canopy of the trees. I started with section 19.2.1 and filled the FDS input file with information as needed, from other FDS examples.  

**Input FDS file**
This is my FDS input file
[forest.txt](https://github.com/firemodels/fds/files/14403895/forest.txt)

**Expected behavior**
I would like to see the tree particles in smokeview. In addition, I would like to visualise the deceleration of the velocity of the flow due to the presence of trees.  

**What I get**
I get the velocity field slice, but there are no trees in the simulation. Therefore I am not able to understand at which point the velocity is influenced by the presence of trees 

**Desktop:**
 - OS: MacOS
 - Version: Sonoma 14.2.1

**Comment by rmcdermo** [comment]
Did you load particles in Smokeview?

**Comment by rakyuv** [comment]
Yes, I loaded the particles in Smokeview. I couldn't see anything. 

On the other hand, I had run the examples 'pine_needles.fds' and 'pine_tree.fds' and I was able to see the particles with Smokeview

**Comment by ericvmueller** [comment]
I can take a look if you want @rmcdermo

**Comment by rmcdermo** [comment]
@ericvmueller Great.  Thanks

**Comment by rakyuv** [comment]
Thank you very much @ericvmueller for your answer. I have added the ```SURF_ID='forest'``` in the definition of VENT corresponding to ```'ZMIN'```. And I have removed branches from ```INIT```, and reduced ```INIT_PER_AREA``` to 0,01. 

I can now see the trees in Smokeview.

I would like to ask something about the aspect ratio of the cells. If I maintain the cells to be of cubic shape thereby maintaining the aspect ratio, would it matter too much if my domain is longer on one axis compared to the other two axes ?
